images/sles-sap/azure-li/15-sp3/image.yaml

```diff
@@ -0,0 +1,8 @@
+include-paths:
+  - sle15/sp1
+  - sle15/sp2
+  - sle15/sp3
+image:
+  name: SLES15-SP3-SAP-Azure-LI-BYOS
+  specification: "SUSE Linux Enterprise Server for SAP Applications 15 SP3 Azure LI BYOS guest image"
+  version: 1.0.42
```

Comment:
While not in production version number should be 0.9.x see the "Image Versioning" document in our confluence area.